.github/workflows/deploy-gcp-tests.yml

```diff
@@ -274,6 +294,62 @@ jobs:
           --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} \
           ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}"
 
+      # SSH into the just created VM, and create a Docker container to run the incoming test 
+      # from ${{ inputs.test_id }}, then create a docker volume with the recently attached disk.
+      # The disk will be mounted in ${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }},
+      # and ${{ inputs.root_state_path }}/${{ inputs.lwd_state_dir }}
+      #
+      # In this step we're using the same disk for simplicity, as mounting multiple disks to the
+      # VM and to the cointainer might require more steps in this workflow, and additional
+      # considerations.
+      #
+      # The disk mounted in the VM is located at /dev/sdb, we mount this disk to the docker
+      # container in two different paths:
+      # - /var/cache/zebrad-cache -> ${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} -> $ZEBRA_CACHED_STATE_DIR
+      # - /var/cache/lwd-cache -> ${{ inputs.root_state_path }}/${{ inputs.lwd_state_dir }} -> $LIGHTWALLETD_DATA_DIR
+      #
```

Comment:
```suggestion
      #
      # This doesn't cause any path conflicts, because Zebra and lightwalletd create different
      # subdirectories for their data. (But Zebra, lightwalletd, and the test harness must not
      # delete the whole cache directory.)
      #
```